### ISSUE TITLE
Report Advertisement in Group Finder not working/tainting

### ISSUE THREAD

**Issue by Trenchfoote** [tukui-org/ElvUI#1291]
### Troubleshooting Steps

Edebug enabled, open LFG, find an advertisement and attempt to report. Report frame does not show and a taint is populated.
![image](https://github.com/user-attachments/assets/3032757f-f873-47d8-9002-1ab1c6ed01bd)

### Describe the expected behavior and what actually happened?

I expect to be able to report an advertisement so, at least, they're not showing in group finder anymore.

In basic Blizz UI, no errors are shown and the message "Complaint Registered" is printed.

### Reproducing the issue.

Enable edebug (/edebug on)
Open Group finder and navigate to premade groups, then dungeons
Find an advertisement and attempt to report
Report frame will not show

### Lua Errors

```shell
2x [ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'SendReport()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `SendReport'
[string "@Blizzard_ReportFrameShared/ReportFrameShared.lua"]:194: in function `SendReport'
[string "@Blizzard_ReportFrameShared/ReportFrameShared.lua"]:102: in function `InitiateReportInternal'
[string "@Blizzard_ReportFrameShared/ReportFrameShared.lua"]:472: in function <...Ons/Blizzard_ReportFrameShared/ReportFrameShared.lua:458>
[string "=[C]"]: in function `SetAttribute'
[string "@Blizzard_ReportFrameShared/ReportFrameShared.lua"]:71: in function `InitiateReport'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:3884: in function `LFGList_ReportAdvertisement'
[string "@Blizzard_GroupFinder/Mainline/LFGList.lua"]:3153: in function <...aceBlizzard_GroupFinder/Mainline/LFGList.lua:3152>
[string "=(tail call)"]: ?
[string "=[C]"]: in function `securecallfunction'
[string "@Blizzard_Menu/Menu.lua"]:854: in function `Pick'
[string "@Blizzard_Menu/MenuTemplates.lua"]:64: in function <Blizzard_Menu/MenuTemplates.lua:58>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
}
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by Trenchfoote** [comment]
Couple of additional notes:

- This was replicated on a fresh log in.
- Taint shows 2 errors, first is the GetPlayerString taint
- (Odd one) Premade Groups Filter allows the report, but is doing something weird (forces a menu; BlizzUI just reports)

**Comment by kodewdle** [comment]
first one, please read: https://github.com/tukui-org/ElvUI/issues/1225#issuecomment-2253700369
second one may be a direct reflection of the first, perhaps.

**Comment by Trenchfoote** [comment]
Comment suggests something resolved it in an 11.0.2 build? I'll wait until next week and test again.
